packages/components/src/components/icon/common/ic-not-applicable-verification-default.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Did you run SVGO on the icons to minify them?

Reply:
@matin-deriv i missed optimization of few icons.. i ran svgo on all icons again and pushed to this PR. 👍